src/javascript/EditPanel/EditPanel.jsx

```diff
@@ -113,7 +113,7 @@ const EditPanelCmp = ({formik, title, classes, notificationContext, client}) =>
                                     return <Button context={context}/>;
                                 }}
                             />
-                            {title}
+                            {title.slice(0, 100) + '...'}
```

Comment:
What happen if there is less than 100char to the title ?

I think you can do unit test about that + handle the case when title is less than 100char

Reply:
"florent".slice(0,100) => "florent"

And it will be updated in the "header style" story with much better handling ...

Reply:
`{title.slice(0, 100) + '...'}` if title is florent, then it will write `florent...`, not `florent`. It's strange isn't it ?